src/azure-cli-testsdk/azure/cli/testsdk/patches.py

```diff
@@ -3,6 +3,8 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
+from azure_devtools.scenario_tests.patches import patch_time_sleep_api, _mock_in_unit_test
```

Comment:
It is used here; almost every patch uses it. But my impression is that it's an internal helper for patches, not something that consumers of azure-cli-testsdk should be using, and thus shouldn't be exposed.